python/pyspark/ml/torch/distributor.py

```diff
@@ -144,15 +145,21 @@ def __init__(
         num_processes: int = 1,
         local_mode: bool = True,
         use_gpu: bool = True,
+        spark: Optional[SparkSession] = None,
     ):
+        if spark is None:
+            self.spark = SparkSession.getActiveSession()
```

Comment:
On spark connect client, we haven't supported `getActiveSession` ? But we can use `_active_spark_session` instead